civismlext/preprocessing.py

```diff
@@ -148,14 +149,25 @@ def __init__(self,
         self.fill_value = fill_value
         self.dataframe_output = dataframe_output
 
+    def _flag_nulls(self, X, cols_to_drop):
+        null_cols = [col for col in X if
+                     X[col].isnull().all() and col not in cols_to_drop]
+        if len(null_cols) > 0:
+            warnings.warn('The following contain only nulls and '
+                          'will be dropped: ' + str(null_cols),
+                          UserWarning)
+        cols_to_drop.extend(null_cols)
```

Comment:
This modifies the input list. I think you should pick either modifying the input or returning a list, but not both. You could return `cols_to_drop + null_cols` and the output will be the same, without modifying the input.